client/src/components/QuoteOfTheDay/SwitchCategory/style.module.scss

```diff
@@ -0,0 +1,43 @@
+.container {
+    display: flex;
+    flex-direction: column;
+    width: 340px;
+    height: fit-content;
+    background-color: #ffffff;
+    border-bottom-left-radius: 25px;
+    border-bottom-right-radius: 25px;
+    box-shadow: 0px 4px 16px 8px rgba(0, 0, 0, 0.1);
+}
+
+.listHolder {
+    padding-top: 1vh;
+    overflow-y: auto;
```

Comment:
What's the purpose of this?

Reply:
This is like a container for the list. 
Pretty much the parent element of all the options that the user can select from.
Did this so that only a section of the actual widget is like scrollable as opposed to the whole widget being scrollable/filled with option